ani-cli

```diff
@@ -130,6 +130,13 @@ get_links() {
 |g' | sed -nE 's|.*link":"([^"]*)".*"resolutionStr":"([^"]*)".*|\2 >\1|p;s|.*hls","url":"([^"]*)".*"hardsub_lang":"en-US".*|\1|p')"
 
     case "$episode_link" in
+	*repackager.wixmp.com*)
+            extract_link=$(printf "%s" "$episode_link" | cut -d'>' -f2 | sed 's|repackager.wixmp.com/||g;s|\.urlset.*||g')
+            for j in $(printf "%s" "$episode_link" | sed -nE 's|.*/,([^/]*),/mp4.*|\1|p' | sed 's|,|\
+|g'); do
```

Comment:
What's going on here?

Reply:
The for loop above this sorts the links in order of decreasing quality and prints them

Whereas these lines print another .mp4 link from wixmp to the console

That sounded like an interview lol

Reply:
I meant why is there a new line in the expression, sorry for being so inaccurate.
Can't we do a \n or otherwise avoid this thing spanning two lines?

Reply:
You could try tho, should theoretically be possible

Reply:
Because we're the ones who have to tend to this codebase, and I don't want to have a hard time because of some syntax party trick.

It's not getting merged until shfmt is appeased.

Reply:
I checked the log, shfmt complains about something else, still the literal newline is not approved by me.

Reply:
> I checked the log, shfmt complains about something else, still the literal newline is not approved by me.

so what do you want me to do?

Reply:
> > I checked the log, shfmt complains about something else, still the literal newline is not approved by me.
> 
> so what do you want me to do?

See if it works with \n instead of the literal newline

Reply:
No, I think we put the similar code cause of not breaking BSD compatibility.

Reply:
What about referring to the character with its hexa/octal value?

Reply:
I think I remember the sed commands needed an actual new line instead of a "\n" character to work on BSD.

Reply:
Sorry for being rude about it then